Added Hello file for Introduction task

TamariSadu opened the pull request.

NodarNoz commented:
Thanks for update, but it still is not perfect. you have extra files on both branches.
You should not have Hello.txt file on mine branch. 
On Introduction_task Branch you have three files, you only need one.
Also Please Rename Repo as it was requested in Task description.
I think It will be useful if you revisit course [[Version Control with Git | Learn]](https://learn.epam.com/catalog/detailsPage?id=16d7f2e7-cc80-4870-928e-400723f732bb), that will help you to better understand how to properly use git.

TamariSadu commented:
I have cleaned up the files as per the task requirements.
Removed unnecessary files from the main branch.
Kept only Hello.txt in the Introduction_Task branch.
Renamed the repository to Tamari_Sadunishvili_SP25 as instructed.
Please review the changes.

NodarNoz commented:
Thanks for update, 
But it still has one conceptual mistake.
**Removed unnecessary files from the main branch.** - Hello.txt should not be there, what's the point of pull request if you already added your work to main? Please remove Hello.txt from main.


Good Git practice is to create files in a branch first, then merge them into main through a pull request. This keeps things clean, avoids confusion, and makes collaboration easier. 
For next task you'll create new branch and  add your work **only there** and create pull request, this way you will have dedicated branch for each task and all the tasks will be merged to main using pull request approved by mentor.
![Screenshot 2025-03-14 134657](https://github.com/user-attachments/assets/642fe5b7-7edb-427d-b5f2-bbf5716efaf4)
